android/src/main/java/com/mapbox/mapboxgl/MapboxMapController.java

```diff
@@ -558,6 +558,11 @@ private Feature firstFeatureOnLayers(RectF in) {
 
   @Override
   public void onMethodCall(MethodCall call, MethodChannel.Result result) {
+    String annotationManagerNotCreatedErrorCode = "NO ANNOTATION MANAGER";
+    String annotationManagerNotCreatedErrorMessage = "To use %ss please add it to the annotation list";
```

Comment:
We can make these static final constants, this avoid allocating strings with every method call